modules/webhook-configuring-acs.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-using-generic-webhooks.adoc
+:_module-type: PROCEDURE
+[id="webhook-configuring-acs_{context}"]
+= Configuring {product-title}
+
+[role="_abstract"]
+Create a new integration in {product-title} by using the webhook URL.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Notifier Integrations* section and select *PagerDuty*.
+. Click *New Integration* (add icon).
```

Comment:
Noting that `(add icon)` will need to be updated before merge. (Similar comment for the last 2 lines of the PR.)

Reply:
Previously, I've used it like this. PS: https://github.com/openshift/openshift-docs/pull/33863#discussion_r658370935

I'm open to other suggestions if we can make it better.

Reply:
Oh, I see. That makes sense, thanks. I thought it was a reminder to actually add the icon image later. :)

Reply:
Actually, I see in another PR that you format "Add" in bold, so maybe that's better? https://github.com/openshift/openshift-docs/pull/33962/files#diff-1be27dd197b5fbbdb65f876bd9a8624b3dc867d49d8610d8072ea45fe2553310R19

Reply:
Thanks @bobfuru Works for me. 

I'm changing all similar instances to the following:
```
(*`add`* icon)
```